.github/workflows/release.yaml

```diff
@@ -37,11 +37,12 @@ jobs:
         with:
           fetch-depth: 0
       - name: Select charmhub channel
-        uses: canonical/charming-actions/channel@2.1.0
+        uses: canonical/charming-actions/channel@2.2.2
         id: channel
       - name: Upload charm to charmhub
-        uses: canonical/charming-actions/upload-charm@2.1.0
+        uses: canonical/charming-actions/upload-charm@2.2.2
         with:
+          destructive-mode: false
```

Comment:
let's remove this like to close PR and move forward with charm focal+jammy when we can.